src/components/JobConfiguration.vue

```diff
@@ -10,13 +10,13 @@
       <ion-item>
         <ion-icon slot="start" :icon="timeOutline" />
         <ion-label>{{ $t("Run time") }}</ion-label>
-        <ion-label id="open-run-time-modal" slot="end">{{ job?.runTime ? getTime(job.runTime) : $t('Select run time') }}</ion-label>
+        <ion-label @click="setOpen(true)" slot="end">{{ job?.runTime ? getTime(job.runTime) : $t('Select run time') }}</ion-label>
```

Comment:
Add a TODO that we are facing issues with trigger for modal.
We could directly set isModalOpen to true

Reply:
As checked, the issue is because we have JobConfiguration component twice. Use v-if for desktop only component

Reply:
Sir, updated code.